submissions/okkkko/js_oop/app.js

```diff
@@ -3,13 +3,13 @@ const STEP_Y = 80;
 const STEP_X = 102;
 const BUGS_PADDING = 18;
 const BUGS_START_POSITION = {
-  x: -200,
-  y: [1, 3, 4].map((rowNumber) => rowNumber * STEP_Y - BUGS_PADDING),
+  x: -200
 };
```

Comment:
If this just X coordinate why do you need store it in object ?
And it is not *bugs* this is *bug* variable (single *bug*) So, change naming.